src/test/java/cz/czechitas/Page.java

```diff
@@ -0,0 +1,31 @@
+package cz.czechitas;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.*;
+
+public class Page {
+    WebDriver driver;
+
+    public Page(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public void login(String loginPassword, String loginUserName){
+        WebElement user = driver.findElement(By.id("username"));
```

Comment:
Neni to nezbytne nutne, ale casto se ty By selectory vytahuji do jako promenna na uroven tridy. Ten kod je potom totiz o neco malo prehlednejsi.